sdk/src/Core/Amazon.Runtime/CredentialManagement/NetSDKCredentialsFile.cs

```diff
@@ -63,6 +63,7 @@ public class NetSDKCredentialsFile : ICredentialProfileStore
 
         private const string SsoAccountId = "sso_account_id";
         private const string SsoRegion = "sso_region";
+        private const string SsoRegistrationScopes = "sso_registration_scopes";
```

Comment:
Can you add this to `ReservedPropertyNames` alongside the others for SSO?

Reply:
Done